docs/topics/reading-and-writing-to-file.md

```diff
@@ -703,6 +703,31 @@ echo $writer->generateSheetData();
 echo $writer->generateHTMLFooter();
 ```
 
+#### Editing HTML During Save Via a Callback
+
+You can also add a callback function to edit the generated html
+before saving. For example, you could add a webfont
+(not currently supported for Pdf) as follows:
```

Comment:
We should find a better example than webfont, because they are actually useless as of now. It does not make much sense to demonstrate something that has no purpose.

Reply:
I will certainly make the other changes you recommend, but I'm having trouble understanding your statements about webfonts. Do you find them useless in general? Or useless for this application? Or useless for this application because they can't be used for PDF? Knowing your particular objection will make it easier for me to find a better example. On the subject of which, would simply adding fallback generic fonts qualify as better?

Reply:
I have tried something else, hopefully more to your liking.